src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+
+  const declarations = sourceString.split(';');
+
+  for (let declaration of declarations) {
+    // Trim spaces from each declaration
+    declaration = declaration.trim();
+
+    if (declaration === '' || !declaration.includes(':')) {
+      continue;
+    }
+
+    const [key, value] = declaration.split(':');
+
+    styleObject[key.trim()] = value.trim();
```

Comment:
And after remove here
```suggestion
    styleObject[key] = value;
```